Generator/Tests/RenderDeprecatedAttributeTests.swift

```diff
@@ -0,0 +1,98 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2025 the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Testing
+
+@testable import Generator
+
+struct RenderDeprecatedAttributeTests {
+    let renderer = OTelAttributeRenderer(
+        context: Context(
+            // Minimal namespace tree to resolve renamed deprecations
+            rootNamespace: Namespace(
+                id: "",
+                subNamespaces: [
+                    "foo": Namespace(
+                        id: "foo",
+                        attributes: [
+                            "bar_baz": Attribute(
+                                id: "foo.bar_baz",
+                                type: Attribute.StandardType.string,
+                            )
+                        ],
+                    )
+                ]
+            )
+        )
+    )
+
+    @Test func testRenamed() {
+        let actual = renderer.renderDeprecatedAttribute(
+            .renamed(renamed_to: "foo.bar_baz", note: "Replaced by a new attribute `foo.bar_baz`"),
+        )
+        #expect(
+            actual
+                == "@available(*, deprecated, renamed: \"OTelAttribute.foo.barBaz\", message: \"Replaced by a new attribute `foo.bar_baz`\")"
```

Comment:
This comes down to personal preference so feel free to choose what you prefer, but I think using raw string literals `#""#` instead of escaping every single quote is slightly more readable.

Reply:
Great point! Fixed in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/19/commits/2948ae6e32230bbbdc9cfb7fadab4d86baf27ee1